### ISSUE TITLE
Outline Text Input Border Bug

### ISSUE THREAD

**Issue by ajoykarmakar459** [callstack/react-native-paper#2644]
When we focused the input the label shift to the top and there we got a border on the left side.
Attached the screenshot -

<img width="504" alt="Screenshot 2021-04-05 at 12 51 34 PM" src="https://user-images.githubusercontent.com/23628008/113548792-cb086b80-960d-11eb-818c-e78ed0353f07.png">

**Dependencies -**

  ```
"dependencies": {
    "@react-native-community/masked-view": "^0.1.10",
    "@react-navigation/native": "^5.9.4",
    "@react-navigation/stack": "^5.14.4",
    "react": "17.0.1",
    "react-native": "0.64.0",
    "react-native-gesture-handler": "^1.10.3",
    "react-native-paper": "^4.7.2",
    "react-native-reanimated": "^2.1.0",
    "react-native-safe-area-context": "^3.2.0",
    "react-native-screens": "^3.0.0",
    "react-native-vector-icons": "^8.1.0"
  }
```

**Comment by lukewalczak** [comment]
Hey @ajoykarmakar459, could you please create a [snack](https://snack.expo.io/) or add code sample to help us with reproducing the bug? Please also specify on which platform you're able to get this bug. Thanks!

**Comment by ajoykarmakar459** [comment]
JSX - 
  ```
const [name, setName] = React.useState('');

   <TextInput
              style={GlobalStyle.input}
              mode={'outlined'}
              label="Name"
              value={name}
              onChangeText={text => setName(text)}
            />
```

STYLE -

 ```
 input: {
    fontSize: 14,
    marginBottom: 16,
    justifyContent: 'center',
    backgroundColor: 'white',
  },
```

**Comment by lukewalczak** [comment]
Actually it's very odd bug, because I wasn't able to catch it on the device, however was able to observe it only on the simulator (pixel 3). Are you able to get that issue on the real device?

**Comment by lukewalczak** [comment]
That issue should be fixed in the latest version (4.8.0) 🤞🏽

**Comment by akhilsanker** [comment]
@lukewalczak ,
Still have the issue in android.

System info:
"react-native-paper": "^4.12.2",
  "react": "17.0.2",
  "react-native": "0.66.2",
  Android 12
  
 Any workaround?
 
 Thanks
